backend/src/middleware-callbacks/MessagesCallbacks.ts

```diff
@@ -43,6 +44,42 @@ class MessageCallbacks {
     })
   }
 
+  public createMessageForUser = async (request: Request<{}, {}, AdminMessageCreate>, response: Response, next: NextFunction) => {
+    const requestBody = request.body
+    const adminRep = await AdminService.getAdmins();
+    const messageCount = await MessageService.getMessagesCount();
+   
+    
+    if (messageCount) {
+      const adminAccountId = messageCount[0].receiverId;
+      const messageEntity = await MessageService.createMessageEntity({ 
+      content: requestBody.content,
+      destinationId: adminAccountId,
+      originId: requestBody.userAccountData.userAccountId });
+
+    return  MessageService.saveMessageEntity(messageEntity).then((message) => {
+        return response.json(message);
+    }).catch((error) => {
+        return response.json({ error });
+    })
+    }
+    if (adminRep){
+      const messageEntity = await MessageService.createMessageEntity({ 
+        content: requestBody.content,
+        destinationId: adminRep[0].accountId,
+        originId: requestBody.userAccountData.userAccountId });
+  
+      return  MessageService.saveMessageEntity(messageEntity).then((message) => {
+          return response.json(message);
+      }).catch((error) => {
+          return response.json({ error });
+      })
```

Comment:
send status error 500 Intenal server error

Reply:
same as below put status code always and please double check all the implemented endpoint by you too